### PR TITLE
[TTS]VITS initialize method, test=tts

### PR DESCRIPTION
**Pull request by WongLaw**
Vits initialize method, test=tts

### CODE REVIEW

File: paddlespeech/t2s/models/vits/vits.py [hunk 1]
```diff
@@ -497,3 +505,40 @@ def voice_conversion(
...
+
+    def reset_parameters(self):
+        def _reset_parameters(module):
+            if isinstance(module, nn.Conv1D) or isinstance(module,
+                                                           nn.Conv1DTranspose):
+                kaiming_normal_(module.weight, mode="fan_out")
+                if module.bias is not None:
+                    fan_in, _ = _calculate_fan_in_and_fan_out(module.weight)
+                    bound = 1 / math.sqrt(fan_in) if fan_in > 0 else 0
+                    uniform_(module.bias, -bound, bound)
```
Comment (yt605155624) [inline paddlespeech/t2s/models/vits/vits.py:517]: 是否可以和 conv1d 放在一起，看起来逻辑是一致的

File: paddlespeech/t2s/models/vits/vits.py [hunk 2]
```diff
@@ -497,3 +505,40 @@ def voice_conversion(
...
+            if isinstance(module, nn.Embedding):
+                normal_(module.weight)
+                if module._padding_idx is not None:
+                    with paddle.no_grad():
+                        module.weight[module._padding_idx] = 0
+            if isinstance(module, nn.LayerNorm):
```
Comment (yt605155624) [inline paddlespeech/t2s/models/vits/vits.py:540]: 是否可以和 batchnorm 放在一起

File: paddlespeech/t2s/models/vits/vits.py [hunk 3]
```diff
@@ -497,3 +505,40 @@ def voice_conversion(
             lids, )
 
         return dict(wav=paddle.reshape(wav, [-1]))
+
+    def reset_parameters(self):
+        def _reset_parameters(module):
+            if isinstance(module, nn.Conv1D) or isinstance(module,
+                                                           nn.Conv1DTranspose):
+                kaiming_uniform_(module.weight, a=math.sqrt(5), mode="fan_out")
```
Comment (yt605155624) [inline paddlespeech/t2s/models/vits/vits.py:513]: mode="fan_out" 不需要吧？

File: paddlespeech/t2s/models/vits/vits.py [hunk 4]
```diff
@@ -214,6 +220,8 @@ def __init__(
         self.reuse_cache_gen = True
         self.reuse_cache_dis = True
 
+        self.reset_parameters()
+
```
Comment (yt605155624) [inline paddlespeech/t2s/models/vits/vits.py:224]: hifigan generator 的初始化
